File: DFN_Sample_Project_(Android_Java)/app/src/main/java/com/igaworks/dfnsampleproject/MyApplication.java
package com.igaworks.dfnsampleproject;

import android.app.Application;

import com.igaworks.v2.core.application.AbxActivityHelper;
import com.igaworks.v2.core.application.AbxActivityLifecycleCallbacks;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // DFN SDK init
        AbxActivityHelper.initializeSdk(this, "your_appkey", "your_secretkey");

        // Add Activity LifeCycle (Required)
        registerActivityLifecycleCallbacks(new AbxActivityLifecycleCallbacks());

    }
}

File: DFN_Sample_Project_(Android_Kotlin)/app/src/main/java/com/igaworks/dfn_sample_projectkotilin/MyApplication.kt
package com.igaworks.dfn_sample_projectkotilin

import android.app.Application
import com.igaworks.v2.core.application.AbxActivityHelper
import com.igaworks.v2.core.application.AbxActivityLifecycleCallbacks

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // DFN SDK init
        AbxActivityHelper.initializeSdk(this, "your_appkey", "your_secretkey")

        // Add Activity LifeCycle (Required)
        registerActivityLifecycleCallbacks(AbxActivityLifecycleCallbacks())

    }
}